File: usage/python/addfriend.py
import requests

def exampleAddFriendSecondary():
    url = 'https://execross.com/api/v3/addfriend'
    
    params = {
        'apikey': 'forexecman',
        'appName': 'DESKTOPMAC\t8.5.2\tMAC\t10.15.7', #Example Desktop win appname: 'DESKTOPWIN\t8.5.0\tWINDOWS\t10.0'
        'authToken': '', #change with authToken V1 or v3
        'proxy': '47.251.70.179',
        'mid': 'ua7dfa3ce0339d1033e83e6059f0419e1'
    }
    
    response = requests.get(url, params=params)
    if response.status_code == 200:
        print(response.json())
    else:
        print(f'Error: {response.status_code}')
        print(response.json())


def exampleAddFriendPrimary():
    url = 'https://execross.com/api/v3/addfriend'
    
    params = {
        'apikey': 'forexecman',
        'appName': 'ANDROID\t13.19.1\tANDROID\t12.3.3772',
        'authToken': 'udd56d5cd664be5ecf20d47238b3a8a3d:aWF0OiAxMDI5OTM2NTkxMDAK..,,', #change with your primary
        'proxy': '47.251.70.179',
        'mid': 'ua7dfa3ce0339d1033e83e6059f0419e1'
    }
    
    response = requests.get(url, params=params)
    if response.status_code == 200:
        print(response.json())
    else:
        print(f'Error: {response.status_code}')
        print(response.json())

# Example
exampleAddFriendSecondary() #call func addfriend with secondary
exampleAddFriendPrimary() #call func addfriend with primary

File: usage/javascript/addfriend.js
const ExecrossAPI = require('./exapi');

const api = new ExecrossAPI();

async function main() {
    try {
        const addParams = {
            appName: 'ANDROID\t13.19.1\tANDROID\t12.3.3772',
            authToken: 'uf331c8819d10fa9182db42a6e26:aWF0OiAxMDI5OTM2MTU2MDAK..',  // change with your primary token
            proxy: '',
            mid: 'uca5cbd3fcf6d4344c8ad21ae54c6f9a3',
        }
        const addData = await api.exampleAddFriendPrimary(addParams);
        
        if (addData.error) {
            console.error('Error:', addData.error);
        } else {
            console.log('AddFriend Data:', addData.data);
        }
    } catch (error) {
        console.error('Unexpected Error:', error);
    }
}


main();

/*
Add Friend Data: {
  creator: 'EXECROSS',
  ip: '182.1.100.234',
  proxy: '',
  result: { message: 'success add mrkeceme75 as friend.' },
  status: 200
}
*/